### ISSUE TITLE
msvd-qa test

### ISSUE THREAD

**Issue by fangz-cs** [tsujuifu/pytorch_violet#6]
Thanks for your great work! But I have some questions about the msvd-qa test split. you use an answer set for msvd-qa, How do you deal with questions whose answers are not in the answer set？Just throw them away? I just throw them away get 11983 qa pairs from the original test file(13157 qa pairs). And I use the finetuned-checkpoint you provided, but get a lower accuracy 0.4554

**Comment by tsujuifu** [comment]
Yes, we remvoe them out. 
There are 5,749 pairs for validation and **11,983 pairs for testing**.

I have re-tested [ckpt_violet_msvd-qa.pt](https://drive.google.com/uc?id=1Qhdeq35o_jdu7QZwA52aWz1Ufq8bBexA) and got **0.463530/0.478620 for validation/testing**.

I am using **PyTorch 1.7.0** and **transformers 4.18.0** with **CUDA 11.0**.
Also, do not forget to **add model.eval()** during the evaluation.

**Comment by fangz-cs** [comment]
ok, thank you very much, i already got the right acc.I indeed forgot to add model.eval()
